### DISCUSSION TITLE
Stopped working

### DISCUSSION THREAD

**Discussion by michaelthompson1991** [mampfes/hacs_waste_collection_schedule#1990] [Q&A]
Waste collection schedule stopped working 

So I had this working for probably 2 months.

Yesterday I noticed I didn’t get notified from my automation and the calendar was still there but had no entires in. So I’ve tried deleting the yaml, the integration and doing everything again.

But same problem, it put the calendar in but again has no entries.

Thanks!

**Comment by 5ila5** [comment]
Sounds like your source did not fetch any entries. I need the source you used, we have nearly 400 different scripts (sources) scraping various service providers.

**Reply by michaelthompson1991** [reply]
I’m not sure what I’m doing. If you could send me steps of what to do I’d be happy to help and get this fixed

**Reply by michaelthompson1991** [reply]
Also apologies! Didn’t read it properly. I’m using the uk and Gateshead council. This one here https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/gateshead_gov_uk.md

**Reply by michaelthompson1991** [reply]
Are we any further with this? Let me know if I need to do anything else 👍🏻

**Comment by 5ila5** [comment]
Fixed by #2004

This will be part of release 1.48.0 or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

(or update the file manually)

**Reply by michaelthompson1991** [reply]
So I can just wait till the update? Will I need to set it up again or will the data come back into my local calendar? Thanks for all the hard work!

**Reply by 5ila5** [reply]
No need to set it up again, just update and restart HA (either now to master version or in a few weeks to 1.48.0)

**Reply by michaelthompson1991** [reply]
No problem, I tried using master but when I clicked it it just went back to 1.47 so I’ll probably just wait
